compiler/env/OMRCompilerEnv.hpp

```diff
@@ -54,7 +54,7 @@ class OMR_EXTENSIBLE CompilerEnv
 
 public:
 
-   CompilerEnv(TR::RawAllocator raw, const TR::PersistentAllocatorKit &persistentAllocatorKit);
+   CompilerEnv(TR::RawAllocator raw, const TR::PersistentAllocatorKit &persistentAllocatorKit, OMRPortLibrary* const omrPortLib=NULL);
```

Comment:
Yes, it's possible to make it mandatory. However, I may need to create a separate PR for that. The reason I do it this way is to avoid breaking OpenJ9. So the order should be this change (OMR) then OpenJ9 change then another change to make it mandatory.

Reply:
OpenJ9 uses `CompilerEnv` with the first two parameters currently.

Reply:
Ok. Please open an issue so that we don't forget.

Reply:
Issue opened to track leftover (https://github.com/eclipse/omr/issues/5247)